src/figuresClasses.ts

```diff
@@ -1,11 +1,102 @@
-export interface Figure {}
+export enum Shape {
+  Triangle = 'triangle',
+  Rectangle = 'rectangle',
+  Circle = 'circle',
```

Comment:
[CODE STYLE] - Enum constant names usually start with a capital letter. Please adjust the `Shape` enum to follow this convention. For example, `Triangle` should be `Triangle`, `Rectangle` should be `Rectangle`, and so on.